internal/provider_test/integration_test.go

```diff
@@ -855,11 +861,7 @@ func TestNegative(t *testing.T) {
 		require.Contains(t, string(out), "Invalid resource type: badresource")
 		vars.providerCfg.Timeouts = nil
 
-		// Temporarily override environment variable NUODB_CP_URL_BASE
-		// so that URL is not specified at all
```

Comment:
The comment is still relevant. It's just describing the negative check that we're doing (empty string is equivalent to unspecified in terms of how we process the config).

Reply:
I was a bit overzealous in cutting that comment. I cut it based on how I interpreted it when I first saw it in a PR.